commons/abis.js

```diff
@@ -48,6 +48,29 @@ const ERC20_BYTES32_ABI = [
   }
 ]
 
+const TOKENS_SWAPPED_EVENT_ABI = {
```

Comment:
You have removed this ABI usage. Don't think it is needed now.

Reply:
thanks, I removed it https://github.com/poanetwork/tokenbridge/pull/224/commits/ec887c7c3ca8bd2f028a01af9ad5908d3f52cdd8